web/src/settings_components.ts

```diff
@@ -1367,9 +1384,12 @@ function should_disable_save_button_for_group_settings(settings: string[]): bool
                 setting_name_without_prefix,
                 "realm",
             );
+        } else if (setting_name === "can_remove_subscribers_group") {
```

Comment:
Would need to update this when we add more stream group-based settings such that we do not need to list all the settings here. Possibly deriving a list from the `GroupPermissionSetting` info received from server.

Reply:
Yeah deriving from `GroupPermissionSetting` is probably the way to go here.